core/lib/spree/testing_support/deny_urls.rb

```diff
@@ -2,8 +2,8 @@
 
 module Spree
   module TestingSupport
-    module BlacklistUrls
-      def setup_url_blacklist(browser)
+    module DenyUrls
```

Comment:
I'd probably call this `BlockUrls` or `BlockedUrls` or something. That's probably even a more explanatory name than the original.